api/routes/assessment_routes.py

```diff
@@ -220,3 +225,26 @@ def assessment_stats_for_fund_round_id(
 
 def get_application_json(application_id):
     return get_application_jsonb_blob(application_id)
+
+
+def get_all_flags_v2_for_application(application_id):
+    current_app.logger.info(f"Get all flags for application {application_id}")
+    flags = get_flags_for_application(application_id)
+    flag_schema = AssessmentFlagSchema()
+    return flag_schema._serialize(flags, many=True)
```

Comment:
This looks like we're using an internet `_serialize` method?

Maybe it's better for us to use the public method?

https://github.com/marshmallow-code/marshmallow/blob/5dc855f61759446b3140e3c3bada5a035eab2e1f/src/marshmallow/fields.py#L319-L344

Reply:
Thanks - it should be `.dump` anyway. Fixed now.